io.github.prateekmedia.appimagepool.yml

```diff
@@ -0,0 +1,42 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+rename-icon: appimagepool
+rename-appdata-file: appimagepool.appdata.xml
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=~/Applications
```

Comment:
From my limited testing with heavily lifted sandbox, it seems to be trying to save files in `XDG_CONFIG_HOME` but misses a slash when concatenating the path and filename, so it ends up as `/var/home/bpiotrowski/.var/app/io.github.prateekmedia.appimagepool/confignvim.appimage` on my system.

I thought the idea was to use `~/Applications` so it's easily discoverable outside flatpak environment?

Reply:
Yes that was the idea, but I don't know why it's saving there.

Reply:
Got it, will fix this immediately.